README.md

```diff
@@ -25,7 +25,7 @@ jobs:
         with:
           fetch-depth: 0
       - uses: ruby/setup-ruby@v1
-      - uses: planningcenter/balto-brakeman@v0.5
+      - uses: planningcenter/balto-brakeman@v0
```

Comment:
This is so strange to see.  How do you feel about bumping to v1 just for funsies?  I'm happy to do that in another PR, but I think it's safe to say this is action is in the "stable release" realm.

Reply:
Works for me! I had thought about doing that, but wanted to stay focused on getting all these updates out first.